src/Models/News.php

```diff
@@ -18,7 +18,7 @@
  * @property string                        $text
  * @property bool                          $is_meeting
  * @property bool                          $is_pinned
- * @property bool                          $is_important
+ * @property bool                          $is_highlighted
```

Comment:
Normally those should be re-arranged to stay in alphabetic order but thats not the case here anyway